### PR TITLE
(docs) expand more on the "default" field for parameters

### PR DESCRIPTION
**Pull request by schristoff**

### CODE REVIEW

File: docs/content/wiring.md [hunk 1]
```diff
@@ -51,6 +51,15 @@ You can also provide any other attributes, as specified by the CNAB [parameters]
   default: "wordpress"
 ```
 
+If using the default parameter field it must be set. However, it must be passed in as an empty
```
Comment (ludfjig) [inline docs/content/wiring.md:54]: it's not clear to me what these two sentences mean, maybe could be clearer?

Comment (schristoff) [reply docs/content/wiring.md:54]: Sure! Is there an exact part that is confusing to you?

Comment (ludfjig) [reply docs/content/wiring.md:54]: What does "it must me set", and why you gotta set them to some empty value

Comment (arschles) [reply docs/content/wiring.md:54]: how about the following:

>If you decide to use the default parameter field, you must set it as the empty value

does this capture everything you want to say?

**Review by arschles — CHANGES_REQUESTED** [review]

File: docs/content/wiring.md (hunk 1 above)
Comment (ludfjig) [reply docs/content/wiring.md]: I still dont understand why it has to be empty. Isn't this in contradiction with the above example? 
- name: database_name
  type: string
  default: "wordpress"

**Pull request merged**